### PR TITLE
Add silent mode

### LINKED ISSUE #1: Silent mode
Add an option to disable process.stdout.write.

### PR DESCRIPTION
**Pull request by RolfKoenders**
Add option to disable writing to stdout.
This fixes #1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "npm-install-package",
-  "version": "1.0.2",
+  "version": "1.1.2",
```
Comment (yoshuawuyts) [inline package.json]: could you revert the version bump? - on release I'll be bumping this. Thanks!

**Review by yoshuawuyts — CHANGES_REQUESTED** [review]

**Comment by yoshuawuyts** [comment]
Thanks heaps for the PR; just a lil note and should be good to merge

File: package.json (hunk 1 above)
Comment (RolfKoenders) [reply package.json]: Ah oke! Fixed it.

**Review by yoshuawuyts — APPROVED** [review]

**Comment by yoshuawuyts** [comment]
Released as `v1.1.0` :tada: - thanks heaps!

**Comment by yoshuawuyts** [comment]
Added as maintainer on GitHub and npm too :sparkles:

**Comment by RolfKoenders** [comment]
Cool! Appreciated!

**Pull request merged**